src/Weather/PCMet/Images.cpp

```diff
@@ -154,7 +163,6 @@ PCMet::DownloadLatestImage(const char *type, const char *area,
     if (!runner.Run(job2))
       return Bitmap();
   }
-
```

Comment:
Why?

Reply:
This can crash XCSoar due to TOCTOU bug.

Reply:
Still problem still exists

Reply:
I didn't know anything about this bug - I didn't changed here the old code. But I will look for this Bug - and hopefully I can solve!

Reply:
You added an `assert()`.

Reply:
'Time-of-Check-to-Time-of-Use-Problem' - ok, I will check it

Reply:
Ok, I changed it. But why? I had learned that an assert is operative only in Debug mode (With NDEBUG its only 0)?

Reply:
True, and it will crash only debug builds, but it's still a debug.